functests/utils/reporter.go

```diff
@@ -0,0 +1,67 @@
+package utils
+
+import (
+	"os"
+
+	perfUtils "github.com/openshift-kni/performance-addon-operators/functests/utils"
+	ptpUtils "github.com/openshift/ptp-operator/test/utils"
+	sriovNamespaces "github.com/openshift/sriov-tests/pkg/util/namespaces"
```

Comment:
This is not good we should not use the sriov-tests any more.

sriov-tests is only for the 4.4 branch